packages/babel-traverse/src/path/family.js

```diff
@@ -17,6 +17,42 @@ function addCompletionRecords(path, paths) {
   return paths;
 }
 
+function completionRecordForSwitch(cases, paths) {
+  for (const switchcase of cases) {
+    const consequentLength = switchcase.get("consequent").length;
+    if (consequentLength === 0) {
+      continue;
+    }
+    const isDefaultCase = switchcase.get("test").type === null;
+    if (isDefaultCase) {
+      paths = addCompletionRecords(
+        switchcase.get("consequent")[consequentLength - 1],
+        paths,
+      );
+    } else {
+      const lastCaseStatement = switchcase.get("consequent")[
+        consequentLength - 1
+      ];
+      const hasBreakStatement =
```

Comment:
as latter IfStatements are both checking against `hasBreakStatement` you could just continue here for `!hasBreakStatement` and remove those checks below

also the mentioned IfStatements should form if/else imho

Reply:
Done